Update build, close streams in case of exception

elevenfive opened the pull request:
- Add support for Glide 3.7.0
- Update to Android Gradle Plugin 2.1.0
- Use Support Library 23.4.0
- Use Compile SDK 23
- Use Build Tools 23.0.3
- Use Gradle 2.13

eveliotc commented on scissors/src/main/java/com/lyft/android/scissors/Utils.java:
```diff
@@ -64,17 +65,25 @@ public static Bitmap asBitmap(Drawable drawable, int minWidth, int minHeight) {
+                        }
+                    } catch (Exception e) {
+                        Log.e(TAG, "Error attempting to save bitmap.", e);
+                    }
```
Please create `closeQuietly` method

elevenfive replied:
done, ty for feedback!

eveliotc commented on scissors/src/main/java/com/lyft/android/scissors/Utils.java:
```diff
@@ -92,13 +101,18 @@ public void run() {
+                            outputStream.close();
+                        } catch (Exception e) {
+                            Log.e(TAG, "Error attempting to save bitmap.", e);
+                        }
```
Please use `closeQuietly` method here as well

eveliotc commented:
@elevenfive thanks for this, as per contributing guidelines [can you please sign Lyft's CLA?](https://oss.lyft.com/cla)

elevenfive commented:
Done

elevenfive committed:
Update build, close streams in case of exception

- Add support for Glide 3.7.0
- Update to Android Gradle Plugin 2.1.0
- Use Support Library 23.4.0
- Use Compile SDK 23
- Update travis.yml to use new build components
- Update to Gradle 2.13

eveliotc commented:
Awesome thanks!
